src/lando/main/scm/hg.py

```diff
@@ -668,3 +668,38 @@ def clean_repo(self, *, strip_non_public_commits: bool = True):
                 self.run_hg(["strip", "--no-backup", "-r", "not public()"])
             except HgException:
                 pass
+
+    def merge_onto(
+        self, commit_message: str, target: str, strategy: Optional[MergeStrategy]
+    ) -> str:
+        """Create a merge commit on the specified repo.
+
+        Use the specified `MergeStrategy` if passed. Otherwise, perform
+        a normal merge and fail if there are merge conflicts.
+
+        Return the SHA of the newly created merge commit.
+        """
+        if strategy == MergeStrategy.Ours:
+            # Create a fake `hg debugsetparent` merge.
+            self.run_hg(["debugsetparent", ".", target])
```

Comment:
Is this a debug feature? Seems odd to use it in production.

Also the command is `debugsetparents` not `debugsetparent` (even though both will work due to autocomplete.)

Reply:
Updated to use the proper command.

We currently use this in merge day automation to perform a "fake merge" that replicates the `theirs` merge strategy.